src/lib/Actions.js

```diff
@@ -44,6 +51,31 @@ export function setUserInfo(u, w3, id, info, orgInfo) {
     networkId = id;
     userInfo = info;
     originalUserInfo = orgInfo;
+    checkForUnlockedEth()
+}
+
+export function checkForUnlockedEth() {
+    debugger
+    if(userInfo && userInfo.bCdpInfo && userInfo.bCdpInfo.unlockedEth){
+        const eth = userInfo.bCdpInfo.unlockedEth.toFixed(4)
+        const msg = `your ETH vault auction is completed. you have ${eth} ETH to claim`
+        const btn = 'claim'
+        const btnAction = claimUnlockedEth
+        EventBus.$emit('app-alert', msg, btn, btnAction)
+    }
+}
+
+export async function claimUnlockedEth() {
+    debugger
+    try{
+    EventBus.$emit('app-alert', 'claim unlocked ETH pending')
+    const action = B.claimUnlockedCollateral(web3, networkId, userInfo.proxyInfo.userProxy, userInfo.bCdpInfo.cdp, originalUserInfo.bCdpInfo.unlockedEth)
+    await ApiAction(action, user, web3, 0)
+    refreshUserInfo()
+    }catch (err){
+        checkForUnlockedEth()
```

Comment:
this will probably not do anything, as we will not read user info again, so is it needed?